src/compiler/crystal/tools/doc/type.cr

```diff
@@ -740,12 +764,12 @@ class Crystal::Doc::Type
       types: types,
       # instance_methods: instance_methods,
       # class_methods: class_methods,
-      macros:    macros,
-      constants: constants,
-      # included_modules: included_modules,
-      # extended_modules: extended_modules,
-      # subclasses: subclasses,
-      # including_types: including_types,
+      macros:           macros,
+      constants:        constants,
+      included_modules: included_modules_name_array,
+      extended_modules: extended_modules_name_array,
+      subclasses:       subclasses_full_name_array,
+      including_types:  including_types_name_array,
```

Comment:
I don't think we need these methods. I don't like the `array` in their name (bad for the same reasons that hungarian notation is bad), and `included_modules.map(&.full_name)` is short enough.